src/app/lib/config/services/afaanoromoo.js

```diff
@@ -164,7 +164,7 @@ export const service = {
       hasMostRead: true,
     },
     radioSchedule: {
-      hasRadioSchedule: false,
+      hasRadioSchedule: true,
```

Comment:
I've just realised that we likely need another toggle in these service config files. One that says `onFrontPage` or something along those lines, so this can be checked prior to deciding to render radio schedules on the front page. Otherwise all of these other services will also have schedules on their front pages. (e.g. Afaan Oromoo https://www.test.bbc.com/afaanoromoo). We could do this in a separate PR, though. 

```suggestion
      hasRadioSchedule: true,
      onFrontPage: false,
```

Reply:
Sure, we'll need to get a list all the services that won't have radio-schedules on their frontpages.

Reply:
@sareh I've created a [new issue](https://github.com/bbc/simorgh/issues/6021) for this.